### DISCUSSION TITLE
color-code output logs and setting for

### DISCUSSION THREAD

**Discussion by jamland** [heyAyushh/vscode-solana#9] [Ideas]
I have a suggestion. ​would be cool if anchor output will open on commands or maybe add such a setting. I've spent some time trying to understand how to find it.

Also, would be cool to see output colorized similar to terminal. it is very useful to see errors/success tests

**Comment by heyAyushh** [comment]
We can add a setting if this get votes, but you can always keep the anchor output tab open and all the commands will log.

currently we only force show logs when some commands fail. 

Colorcoding is great but will have to check if it can happen on vscode
